Kitodo/src/main/java/org/kitodo/production/forms/TemplateForm.java

```diff
@@ -349,15 +352,18 @@ public void setTemplateById(int id) {
      */
     public void loadTemplate(int id) {
         try {
-            if (id != 0) {
+            Template flashTemplate = (Template) FacesContext.getCurrentInstance()
+                    .getExternalContext().getFlash().get("duplicatedTemplate");
+            if (flashTemplate != null) {
```

Comment:
Instead of `flashTempltate != null` use `Objects.nonNull(flashTemplate)` as on many places `Objects.nonNull()` checks are already used.